src/documents/ModelTransformationAPI.md

```diff
@@ -2390,7 +2395,7 @@ The transform allows you to specify some ranges, defined by an `upperBound`. The
 
 ### Examples:
```

Comment:
I think this should be h4 rather than h3.

Reply:
I have used my shell syntax (see top-level comment) to review and confirm that all "syntax" and "examples" blocks in the Model Transformation API docs are now at the same level as their sibling.